packages/bot-web-ui/src/components/dashboard/dashboard-components/cards.tsx

```diff
@@ -0,0 +1,82 @@
+//kept sometihings commented beacuse of mobx to integrate popup functionality here
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import LoadModalStore from 'Stores/load-modal-store';
+
+interface CardsProps {
+    load_modal: LoadModalStore;
+}
+
+const Cards = ({ load_modal }: CardsProps) => {
+    const { onDriveConnect } = load_modal;
+    // todo for the file input
+    // const file_input_ref = React.useRef(null);
+    // const [is_file_supported, setIsFileSupported] = React.useState(true);
+
+    const IconArray = [
+        {
+            icon: 'IcMyComputer',
+            content: 'My computer',
+        },
+        {
+            icon: 'IcGoogleDriveDbot',
+            content: 'Google Drive',
+            method: onDriveConnect,
+        },
+        {
+            icon: 'IcBotBuilder',
+            content: 'Bot Builder',
+            //need to use react hooks to switch TODO need to use method created from store
+            method: () => {
+                return document.getElementById('id-bot-builder')?.click();
```

Comment:
why are we using `document.getElementById` 😅

Reply:
done